cli/run.go

```diff
@@ -267,6 +267,9 @@ func (r WorkflowRunner) runWorkflows(tracker analytics.Tracker) (models.BuildRun
 	if err := os.Setenv("BITRISE_TRIGGERED_WORKFLOW_TITLE", targetWorkflow.Title); err != nil {
 		return models.BuildRunResultsModel{}, fmt.Errorf("failed to set BITRISE_TRIGGERED_WORKFLOW_TITLE env: %w", err)
 	}
+	if err := bitrise.SetBuildFailedEnv(false); err != nil {
```

Comment:
Set the initial state of the build status envs and update it after each step run.